.changeset/long-hairs-boil.md

```diff
@@ -0,0 +1,6 @@
+---
+"strapi-cms": minor
+"strapi-provider-upload-custom": minor
```

Comment:
If you are creating a new package, shouldn't be a `major` release?

Reply:
I only followed the [semver](https://semver.org/#summary) specs but i can change them if you prefer.

Reply:
**suggestion**: since you are updating two packages, I'd create two different changesets just to have a meaningful changelog for both packages.